docs/02-for-app-authors/02-requirements.md

```diff
@@ -30,6 +33,95 @@ Every repository must contain a manifest describing how to build the application
 
 This manifest may import other manifest files in the same repository. It's quite common to add the [shared-modules](shared-modules) repository as a submodule.
 
+#### Example manifest
+
+This assumes that upstream brings appdata, icons and desktop files and installs them via their post build install. In this case we're using `meson` as the buildsystem.
+
+<Tabs groupId="manifest-language" defaultValue="json" queryString>
+  <TabItem value="json" label="json" default>
+
+```json title="io.github.example.MyCoolApp.json"
+{
+  "id": "io.github.example.MyCoolApp",
+  // The runtime is the environment that the application will run in, so you can use dependencies it provides
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  // This is the command that will be run when the application is launched
```

Comment:
*And put into the desktop file, no?

Reply:
I guess, but not sure that's relevant here

Reply:
Just helps to clear up a bit of confusion; if someone puts there desktop file as something different from the CLI, they might be a bit bamboozled when they don't see the arguments they put in getting passed.